styleguide/_themes/global/scss/forms.scss

```diff
@@ -37,12 +37,12 @@ table {
   .help-block,
   .radio,
   .radio-inline {
-    color: #eb0022;
+    color: $brand-danger;
   }
 
   .form-control {
     background-color: rgba(235, 0, 34, .1);
```

Comment:
@WidgetsBurritos do we want to have a group chat about this or go ahead and change it? its $brand-danger with an opacity on it. not sure if we wanna set a $brand-danger-bg or something.